src/agent/rustjail/src/mount.rs

```diff
@@ -717,31 +711,28 @@ fn mount_from(
         }
     };
 
-    match stat::stat(dest.as_str()) {
-        Ok(_) => {}
-        Err(e) => {
+    stat::stat(dest.as_str())
+        .map_err(|e| {
             log_child!(
                 cfd_log,
                 "dest stat error. {}: {}",
                 dest.as_str(),
                 e.as_errno().unwrap().desc()
-            );
-        }
-    }
+            )
+        })
+        .ok();
 
-    match mount(
+    mount(
         Some(src.as_str()),
         dest.as_str(),
         Some(m.r#type.as_str()),
         flags,
         Some(d.as_str()),
-    ) {
-        Ok(_) => {}
-        Err(e) => {
-            log_child!(cfd_log, "mount error: {}", e.as_errno().unwrap().desc());
-            return Err(e.into());
-        }
-    }
+    )
+    .map_err(|e| {
+        log_child!(cfd_log, "mount error: {}", e.as_errno().unwrap().desc());
```

Comment:
Do we need to unwrap, etc? Can you just print `e` with `{:?}`?

Reply:
I want to keep the logic same as before in this pr.

Reply:
You could add one extra commit to do this though? That way, we can still bisect cleanly.

Reply:
ah, I just understand what you mean now, I will replace `{}` with `{:?}`.

Reply:
and do you mean to remove the unwrap of `e.as_errno().unwrap()` ?

Reply:
done